### PR TITLE
[DOC] Bids layout clarifications

### PR DESCRIPTION
**Pull request by 36000**
This PR is meant to address the issues raised in #695

### CODE REVIEW

File: docs/source/usage/data.rst [hunk 1]
```diff
@@ -56,6 +57,20 @@ data set in a directory called `stanford_hardi`::
...
+See :ref:`examples/plot_bids_layout` for a more extensive example.
+
+
+Preprocessing
+-------------
+pyAFQ does not perform preprocessing. Instead, pyAFQ expects the outputs of
+a preprocessing pipeline in BIDS format. In the above example, the dataset
+was preprocessed using the `VISTASOFT <https://github.com/vistalab/vistasoft>`_ software.
+The outputs of VISTASOFT are stored in its own pipeline folder,
+`derivatives/vistasoft`. Other examples may use other common preprocessing
```
Comment (36000) [inline docs/source/usage/data.rst:69]: clarify that it doesnt have to be derivatives/vistasoft, it could be derivatives/custompipeline

**Comment by 36000** [comment]
Closes #695

**Comment by 36000** [comment]
@arokem this is ready for review / merge

**Comment by arokem** [comment]
This looks good to me! @jyeatman : could you please look this over, when you get a chance?

**Pull request merged**